src/components/Filter.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+const Filter = ({
+  currentFilters,
+  setCurrentFilters,
+  filterOpen,
+  setFilterOpen,
+}) => {
+  return (
+    <div className="filter-container">
+      {filterOpen && (
+        <div className="option-container">
+          {" "}
+          <>
+            {" "}
+            <label>
+              <input
+                type="checkbox"
```

Comment:
Selecting both top and bottom 10 will not highlight any routes. Do we want to be able to select both? Or just one at a time? If the latter, then we may need a radio group.

Reply:
After meeting discussion we're going to table this for now and keep an eye on it for the future!!